packages/config-transformer/tests/Converter/ConfigFormatConverter/AbstractConfigFormatConverterTest.php

```diff
@@ -26,9 +26,9 @@ protected function setUp(): void
         $this->smartFileSystem = $this->getService(SmartFileSystem::class);
     }
 
-    protected function doTestOutput(SmartFileInfo $fixtureFileInfo): void
+    protected function doTestOutput(SmartFileInfo $fixtureFileInfo, string $prefix = ''): void
```

Comment:
This should not be needed, as it circumvents the path check.
The directory name where the fixtures are should be enough.